src/components/10-atoms/checkbox/ui.test.js

```diff
@@ -142,3 +142,20 @@ test('should be clickable + change state and render parent label', async t => {
   // has parent label
   await t.expect(label).ok();
 });
+
+fixture('Checkbox - Variant tick').page(
+  `${host}/iframe.html?id=atoms-checkbox--checkbox&knob-refId=checkbox-rrp2nppsqui&knob-label=this%20is%20a%20label&knob-name=my-checkbox&knob-variant=tick&knob-checked=true`
+);
+
+test('should show checkbox variant tick', async t => {
+  const getIconBackgroundColor = ClientFunction(() => {
+    const checkbox = document.querySelector('axa-checkbox');
+    checkbox.checked = true;
+    checkbox.disabled = false;
+    return window
+      .getComputedStyle(checkbox.querySelector('.a-checkbox__icon--tick'))
```

Comment:
Is is possible to move that querySelector to the one in line 152? Then it is easier to see which element you want to select.

Reply:
Throws me an error: `TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.`

Reply:
ok then leave it like this

Reply:
Never mind it worked 👍